Python/ceval.c

```diff
@@ -532,8 +532,8 @@ PyEval_EvalFrame(PyFrameObject *f) {
 PyObject *
 PyEval_EvalFrameEx(PyFrameObject *f, int throwflag)
 {
-    PyThreadState *tstate = PyThreadState_GET();
-    return tstate->interp->eval_frame(f, throwflag);
+    PyInterpreterState *interp = _PyInterpreterState_Get();
```

Comment:
Hum, PyThreadState_GET() doesn't check if the result is NULL, whereas _PyInterpreterState_Get() does. Do we need a _PyInterpreterState_GetUnchecked() for performance critical code?